frontend/src/slices/auth/auth.slice.ts

```diff
@@ -54,6 +55,17 @@ const { reducer, actions, name } = createSlice({
       state.authDataStatus = DataStatus.REJECTED;
     });
 
```

Comment:
I was not sure if SignOut action actually needs pending and rejected, but added them purely for the consistence purposes, to make sure signOut action follows the general pattern.